scripts/build/build/targets.py

```diff
@@ -499,22 +499,32 @@ def BuildNxpTarget():
     # boards
     target.AppendFixedTargets([
         TargetPart('k32w0', board=NxpBoard.K32W0),
-        TargetPart('k32w1', board=NxpBoard.K32W1)
+        TargetPart('k32w1', board=NxpBoard.K32W1),
+        TargetPart('rw61x', board=NxpBoard.RW61X)
+    ])
+
+    # OS
+    target.AppendFixedTargets([
+        TargetPart('zephyr', os_env=NxpOsUsed.ZEPHYR).OnlyIfRe('rw61x'),
+        TargetPart('freertos', os_env=NxpOsUsed.FREERTOS).ExceptIfRe('rw61x'),
```

Comment:
would this be clearer:

```python
target.AppendFixedTargets([
        TargetPart('k32w0-freertos', board=NxpBoard.K32W0, os_env=NxpOsUsed.FREERTOS),
        TargetPart('k32w1-freertos', board=NxpBoard.K32W1, os_env=NxpOsUsed.FREERTOS),
        TargetPart('rw61x-zephyr', board=NxpBoard.RW61X, os_env=NxpOsUsed.ZEPHYR),
    ])
```

this shows up nicer as `{k32w0-freertos,k32w1-freertos,rw61x-zephyr}` than `{k32w0,k32w1,rw61x}-{zephyr,freertos}` given that we have a forced either/or on the board.

Reply:
We plan to have the rw61x freertos support in this file in the future. So this platform can have a choice either to use freertos or zephyr, this is why I made it like this.

Reply:
Is there a concrete plan to have the `cross-support` (or "future" part) in the near/medium future?

I am looking if we should apply `You ain't gonna need it` here or not. Experience-wise I see about 80+% of things built "for the future" never happening.

Reply:
it is concrete, we are working on it internally, the goal is to merge all nxp platforms in a single builder.